app/services/sso_service.rb

```diff
@@ -72,6 +72,8 @@ def authn_context
 
   def init_new_user(user_identity, existing_user = nil, multifactor_change = false)
     new_user = User.new(user_identity.attributes)
+    new_user.identity = user_identity
```

Comment:
I'm struggling to find a way to properly test this, but if you put `user_identity.attributes[:loa][:current] == new_user.identity[:loa][:current]` after line 74 you'll see that it's not always true.

Reply:
I'm confused as to why you think this is a problem. The idea is that the new_user is built from the newest set of attributes we've received. It should also only be happening if the latest authn_context results in a attribuets class of IdMe... in other words, multifactor or verification.